src/cli/cmd/resolve.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) struct ResolveSubcommand {
     flake_ref: String,
 
     /// Output the result as JSON displaying the store path plus the original attribute path.
-    #[arg(long)]
+    #[arg(long, env = "FH_JSON_OUTPUT")]
```

Comment:
```suggestion
    #[arg(long, env = "FH_OUTPUT_JSON")]
```

in case we have more outputs later?

Reply:
(some in the next file)